clang/docs/MSVCCompatibility.rst

```diff
@@ -154,3 +154,133 @@ a hint suggesting how to fix the problem.
 As of this writing, Clang is able to compile a simple ATL hello world
 application.  There are still issues parsing WRL headers for modern Windows 8
```

Comment:
"modern Windows 8" is obviously stale :) , but not necessarily relevant to this PR

Reply:
Ya its on my todo list to clean this up and to add more details around ABI and/or name mangling support.